doc/manual-wiki/intro.wiki

```diff
@@ -30,6 +30,7 @@ The documentation for TyXML combinators is provided in <<a_api|module Html5_sigs
 The first thing to understand about TyXML is that for most intents and purposes, it is exactly like Html. As such, the [[https://developer.mozilla.org/en-US/docs/Web/HTML/Element|Html reference]] is still very useful. For each Html elements or attributes, there is a combinator implementing it. The main differences are that you can use OCaml to manipulate the elements and that invalid Html produces a type error.
 
 In this tutorial, we will build the [[https://github.com/ocsigen/tyxml/tree/master/examples/mini_website|Mini website]].
+If you prefer the native Html syntax, you can also use the <<a_manual chapter="ppx"|ppx syntax extension>> and consult the [[https://github.com/ocsigen/tyxml/tree/master/examples/mini_website_ppx|ppx mini website]].
```

Comment:
(No important comments, just nitpicking.)

Capitalize HTML and SVG everywhere?